README.md

```diff
@@ -150,14 +124,18 @@ If `spark.cdm.tokenrange.partitionFile.input` or `spark.cdm.tokenrange.partition
 - Supports migration/validation from and to [Azure Cosmos Cassandra](https://learn.microsoft.com/en-us/azure/cosmos-db/cassandra)
 - Validate migration accuracy and performance using a smaller randomized data-set
 - Supports adding custom fixed `writetime`
+- Track run information (start-time, end-time, status, etc.) in tables (`cdm_run_info` and `cdm_run_details`) on the target keyspace
```

Comment:
curious, why target and not origin? Asking as Astra DB would cost them $s while origin mostly is a DSE/OSS C* on-premise which won't incur cost. I can understand the existing production traffic so maybe this is a evaluate the situation and decide. How about we provide this a configurable way and choose one as default? Thoughts?

Reply:
The thought was that `target` will always have write access whereas `origin` may not have. But I agree about the cost concerns & may be we should keep it configurable. May be a new PR, this one is already too large :( 

On a side note, costs should almost always be negligible (0.1 % or less) compared to the data being moved as this is just metadata related to the run. Although it is possible to have a millions entries related to run metadata, in those scenarios, actual data being moved should be several 100 billions.